commands/Eval.cs

```diff
@@ -67,6 +100,38 @@ public static void Info(IUser client)
                     l.ForEachUser(x => client.Print(Template.Text(Category.Info, 1).Replace("+n", x.Name).Replace("+v", x.Vroom.ToString()).Replace("+i", "linked")));
                 });
         }
+        
+        public static void screen(IUser admin, string name)

```

Comment:
Data path will not always be C
Still assuming sb0t.exe is the application exe name when it can be renamed

Again what would the point of this actually be for an admin to be able to take screen shot of the desktop where the server is running.